flake.nix

```diff
@@ -12,12 +12,10 @@
     # Use upstream when https://github.com/nlewo/nix2container/pull/82 is merged
     n2c.url = "github:shlevy/nix2container/no-Size-on-dir";
 
-    marlowe-plutus.url = "github:input-output-hk/marlowe-plutus";
+    marlowe-plutus.url = "github:input-output-hk/marlowe-plutus?ref=aarch64-darwin";
```

Comment:
Will update when https://github.com/input-output-hk/marlowe-plutus/pull/8 is merged